### DISCUSSION TITLE
Questions about images for training dataset

### DISCUSSION THREAD

**Discussion by dathide** [d8ahazard/sd_dreambooth_extension#276] [Q&A]
When training a concept, do the dataset photos need to be square? Do they need to be jpg/png, or does webp work too?

**Comment by cerega66** [comment]
Yes. The ratio should be 1:1. I used both PNG and JPG. I can't say anything about WEBP.

**Comment by dathide** [comment]
I can confirm that webp images work, and training accepts images without a 1:1 ratio, so I wonder if they are automatically cropped.
